src/components/MemoryGraph.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Line } from 'react-chartjs-2';
+import useInterval from '../hooks/use-interval';
+
+/* this line chart graphs the last 10 data points we've seen,
+and updates the lower and upper bound of the range to be equal
+to the min and max of this dataset, respectively. */
+const numData = 10;
+
+const MemoryGraph = ({ next }) => {
+  const [numArray, addToNumArray] = useState([]);
+  const [min, setMin] = useState(0);
+  const [max, setMax] = useState(2500000);
```

Comment:
what is 2500000? should that be a constant variable with a descriptive name?

Reply:
ill add this